test/scenarios/cloudnative/codemodules/codemodules_test.go

```diff
@@ -21,17 +21,17 @@ func TestMain(m *testing.M) {
 	testEnvironment.Run(m)
 }
 
-func TestCloudNative(t *testing.T) {
+func TestCodeModulesImages(t *testing.T) {
 	testEnvironment.Test(t, CodeModules(t, false))
```

Comment:
😬 
```suggestion
	testEnvironment.Test(t, CodeModulesImage(t, false))
```

Reply:
@0sewa0 `CodeModulesImage` or `CodeModulesImages` !? ))

Reply:
https://github.com/Dynatrace/dynatrace-operator/blob/main/src/api/v1beta1/dynakube/oneagent_types.go#L124

Reply:
fixed in be5b4236